app/api/routers/attributes.py

```diff
@@ -2,12 +2,12 @@
 from pydantic import constr
 
 from .. import crud
-from ..models import CONTROLLED_TERM_REGEX
+from ..models import CONTROLLED_TERM_REGEX, CombinedAttributeResponse
 
 router = APIRouter(prefix="/attributes", tags=["attributes"])
 
 
-@router.get("/{data_element_URI}")
+@router.get("/{data_element_URI}", response_model=CombinedAttributeResponse)
```

Comment:
in `crud`, you mentioned https://fastapi.tiangolo.com/advanced/additional-responses/ in a comment (very helpful). So I would expect that here you do have to differentiate between a full success response:

e.g.
```python
{
        "errors": node_errors,
        "responses": cross_node_results,
        "nodes_response_status": "success",
    }
```

and a partial or complete failure response, e.g.
```python
return JSONResponse(
            status_code=status.HTTP_207_MULTI_STATUS,
            content={
                "errors": node_errors,
                "responses": cross_node_results,
                "nodes_response_status": "partial success",
            },
        )
```

no? maybe I'm missing something about the additional responses or the different types of response objects here.

Reply:
See new commits for comments added to explain this!